### ISSUE TITLE
post-processing net with pre-net??

### ISSUE THREAD

**Issue by DabiaoMa** [Kyubyong/tacotron#60]
I did not find any hints in the original paper that in the post-processing module a pre-net is added before CBHG, so why the author add one in the code 'tacotron/networks.py.decode2' ??

**Comment by candlewill** [comment]
Yes, I think your understanding is right. There should be no prenet in post-processing. Some days ago, I pulled a requests to remove it, https://github.com/Kyubyong/tacotron/pull/52
But as the dimension is not consistent, I have closed it.
